pkg/diskmaker/diskmaker.go

```diff
@@ -124,6 +124,77 @@ func (d *DiskMaker) Run(stop <-chan struct{}) {
 	}
 }
 
+func (d *DiskMaker) createSymLink(deviceNameLocation DiskLocation, symLinkDirPath string) {
+
+	var symLinkSource, symLinkTarget string
+	var isSymLinkedByDeviceName bool
+	if deviceNameLocation.diskID != "" {
+		symLinkSource = deviceNameLocation.diskID
+		symLinkTarget = getSymlinkTarget(deviceNameLocation.diskID, symLinkDirPath)
```

Comment:
What happens when someone updates to a new LSO version with symlinks already present from the old LSO? Looking at the code, does it create new symlinks, ignoring the old ones? What happens to already existing PVs?

Reply:
I don't think new symlinks will be created. Old ones will still be used. We are checking for existing symlinks in line 155 below.

Reply:
I've tested this scenario. Updated the results in the PR description.  Looks good.